bosh-setup/azuredeploy.json

```diff
@@ -89,19 +66,42 @@
     },
     "enableDNSOnDevbox": {
       "type": "bool",
+      "defaultValue": true,
       "allowedValues": [
         true,
         false
       ],
       "metadata": {
-        "description": "A default DNS will be setup in the devbox if it is true. Need a manual reboot to take effect."
+        "description": "A default DNS will be setup in the devbox if it is true."
+      }
+    },
+    "tenantID": {
+      "type": "string",
+      "defaultValue": "TENANT-ID",
+      "metadata": {
+        "description": "ID of the tenant. See https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release/blob/master/docs/template-guide.md"
+      }
+    },
+    "clientID": {
+      "type": "string",
+      "defaultValue": "CLIENT-ID",
+      "metadata": {
+        "description": "ID of the client. See https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release/blob/master/docs/template-guide.md"
+      }
+    },
+    "clientSecret": {
+      "type": "securestring",
+      "defaultValue": "CLIENT-SECRET",
+      "metadata": {
+        "description": "secret of the client. See https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release/blob/master/docs/template-guide.md"
       }
     }
   },
   "variables": {
     "api-version": "2015-05-01-preview",
```

Comment:
For now, I prefer to keep 2015-05-01-preview since there is no time to test the latest API version before GA of Cloud Foundry on Azure.
One or two weeks later, I will create another PR to update the API version. Is that OK for you?